slack_bolt/app/async_app.py

```diff
@@ -359,6 +361,7 @@ async def async_dispatch(self, req: AsyncBoltRequest) -> BoltResponse:
         :param req: An incoming request from Slack.
         :return: The response generated by this Bolt app.
         """
+        starting_time = time.time()
```

Comment:
Debug logs used to track only the time spent in a listener function. Starting here is more accurate.